spec/services/cfe_civil/components/capitals_spec.rb

```diff
@@ -17,26 +23,56 @@
            inherited_assets_value: nil,
            money_owed_value: 0.0)
-    allow(legal_aid_application).to receive(:online_savings_accounts_balance).and_return(savings_balance)
-    allow(legal_aid_application).to receive(:online_current_accounts_balance).and_return(current_balance)
+    allow(legal_aid_application).to receive(:online_savings_accounts_balance).and_return(online_savings_balance)
+    allow(legal_aid_application).to receive(:online_current_accounts_balance).and_return(online_current_balance)
+    create(:savings_amount,
+           legal_aid_application:,
+           offline_current_accounts: offline_current_balance,
+           offline_savings_accounts: offline_savings_balance,
+           partner_offline_current_accounts: offline_partner_current_balance,
+           partner_offline_savings_accounts: offline_partner_savings_balance,
+           joint_offline_current_accounts: offline_joint_current_balance,
+           joint_offline_savings_accounts: offline_joint_savings_balance)
   end
 
   describe ".call" do
-    it "returns json in the expected format" do
-      expect(call).to eq({
-        capitals: {
-          bank_accounts: [
-            { description: "Online current accounts", value: current_balance },
-            { description: "Online savings accounts", value: savings_balance },
-          ],
-          non_liquid_capital: [
-            { description: "Timeshare property", value: "555.55" },
-            { description: "Land", value: "777.77" },
-            { description: "Any valuable items worth more than £500", value: "888.88" },
-            { description: "Interest in a trust", value: "999.0" },
-          ],
-        },
-      }.to_json)
+    context "when invoked with no owner" do
+      it "returns json in the expected format with only the applicant bank values" do
+        expect(call).to eq({
+          capitals: {
+            bank_accounts: [
+              { description: "Current accounts", value: offline_current_balance.to_s },
+              { description: "Savings accounts", value: offline_savings_balance.to_s },
+              { description: "Online current accounts", value: online_current_balance },
+              { description: "Online savings accounts", value: online_savings_balance },
+            ],
+            non_liquid_capital: [
+              { description: "Timeshare property", value: "555.55" },
+              { description: "Land", value: "777.77" },
+              { description: "Any valuable items worth more than £500", value: "888.88" },
+              { description: "Interest in a trust", value: "999.0" },
+            ],
+          },
+        }.to_json)
+      end
+    end
+
+    context "when invoked with partner as owner" do
+      subject(:call) { described_class.call(legal_aid_application, "Partner") }
+
+      it "returns json in the expected format with the partner and joint bank values merged" do
+        expect(call).to match_json_expression({
```

Comment:
Out of interest, why have you used `match_json_expression` here but `eq` above, on line 41?

Reply:
It was a debugging issue, match_json_expression doesn't care about the sequence, I'll revert it to `eq` now to match all the others   :+1:

for reference, given a json block of:
```ruby
{ "foo": "bar", "bar": "baz" }

expect({ "foo": "bar", "bar": "baz" }).to eq({ "bar": "baz", "foo": "bar" }) # will fail
expect({ "foo": "bar", "bar": "baz" }).to match_json_expression({ "bar": "baz", "foo": "bar" }) # will pass, as the key/value pairs match
```